src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -290,3 +306,132 @@ def _convert_meta(self, source: Node, concept: Concept):
                     value=synonym.val,
                 )
             )
+
+    @staticmethod
+    def write_concept_maps(obograph_path: str, outdir: str = None, code_system_id: str = None, verbose=True):
+        """"From an Obograph JSON, convert to SSSOM, then convert to 1+ ConceptMap JSON"""
+        # Vars
+        outdir = outdir or obograph_path
+        outdir = outdir if os.path.isdir(outdir) else os.path.dirname(outdir)
+        # todo: could be more combinations like - or _ obographs
+        code_system_id = code_system_id or os.path.basename(obograph_path)\
+            .replace(".obographs", "").replace(".obograph", "").replace(".json", "")
+        outpath_sssom = os.path.join(outdir, f"{code_system_id}.sssom.tsv")
+
+        # TODO obograph -> sssom
+        #  - needs path to be passed to dump
+        #  - before i continue with this, check how owl-on-fhir does. does it use OAK CLI or Python API?
+        #   ...if CLI, then I need to somehow get the CLI to pass the path down here. maybe not difficult
+        #   ...or maybe i should just do that regardless
+        #   ...how did I call the CLI again? maybe i ned to check my notes or save better notes...
+
+        # todo: is there a way to do this via Python API? would be better
+        # todo: -m metadata.sssom.yml: can I create this file on the fly and pass?
+        command_str = f'sssom parse {obograph_path} -I obographs-json -o {outpath_sssom}'
+        print('Converting: Obographs -> SSSSOM')
```

Comment:
## Broad overview of PR
@cmungall I started this back in April in order to create [FHIR NPM package](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification) output. However now I'm also doing ConceptMap output. That will go along well with this PR, since it involves outputting and zipping up a number of files.

Reply:
Splitting this into a new PR, so this topic is now defunct.